src/ngraph_builder.cc

```diff
@@ -59,6 +59,38 @@ static tf::Status ValidateInputCountMin(const tf::Node* op, size_t count) {
   return tf::Status::OK();
 }
 
+template <typename T>
+static void MakePadding(const std::string& tf_padding_type,
```

Comment:
Make this a static member of the NGraphBuilder class so that you can add unit test to specifically test this function. 

Then add a C++ unit test to ensure the correctness of this function with various corner cases. The layout of the test would be something like this:
1. Call this  function with various combination of inputs and validate the output
2. Call this function with different supported data types.

Reply:
👍